### ISSUE TITLE
assign as a type annotation

### ISSUE THREAD

**Issue by kdsin23** [harisreedhar/AN_BLUEFOX#13]
hi,friend:
when i launched the blender,the consol gave me this message:
```

Warning: class an_Colormix2 contains a property which should be an annotation!
/home/jim/programs/blender-2.83-beta-2020.04.24/2.83/scripts/addons_contrib/animation_nodes/auto_load.py:28
    assign as a type annotation: an_Colormix2.clamp
Warning: class an_DupliInstancer contains a property which should be an annotation!
/home/jim/programs/blender-2.83-beta-2020.04.24/2.83/scripts/addons_contrib/animation_nodes/auto_load.py:28
    assign as a type annotation: an_DupliInstancer.useDisplay
    assign as a type annotation: an_DupliInstancer.useRender
```
if possible ,you may fix it.thank you!

**Comment by harisreedhar** [comment]
> hi,friend:
> when i launched the blender,the consol gave me this message:
> 
> ```
> 
> Warning: class an_Colormix2 contains a property which should be an annotation!
> /home/jim/programs/blender-2.83-beta-2020.04.24/2.83/scripts/addons_contrib/animation_nodes/auto_load.py:28
>     assign as a type annotation: an_Colormix2.clamp
> Warning: class an_DupliInstancer contains a property which should be an annotation!
> /home/jim/programs/blender-2.83-beta-2020.04.24/2.83/scripts/addons_contrib/animation_nodes/auto_load.py:28
>     assign as a type annotation: an_DupliInstancer.useDisplay
>     assign as a type annotation: an_DupliInstancer.useRender
> ```
> 
> if possible ,you may fix it.thank you!

It can be fixed by broviding annotations, thanks for the info

**Comment by harisreedhar** [comment]
> hi,friend:
> when i launched the blender,the consol gave me this message:
> 
> ```
> 
> Warning: class an_Colormix2 contains a property which should be an annotation!
> /home/jim/programs/blender-2.83-beta-2020.04.24/2.83/scripts/addons_contrib/animation_nodes/auto_load.py:28
>     assign as a type annotation: an_Colormix2.clamp
> Warning: class an_DupliInstancer contains a property which should be an annotation!
> /home/jim/programs/blender-2.83-beta-2020.04.24/2.83/scripts/addons_contrib/animation_nodes/auto_load.py:28
>     assign as a type annotation: an_DupliInstancer.useDisplay
>     assign as a type annotation: an_DupliInstancer.useRender
> ```
> 
> if possible ,you may fix it.thank you!

Hi bro, I have fixed this issue in 9e5b299d9ba2b088f610e4aa774923a63a0a4c8f . Thanks a lot for the info
